app/pb_kits/playbook/pb_line_graph/line_graph.rb

```diff
@@ -1,11 +1,12 @@
 module Playbook
   module PbLineGraph
-    class LineGraph
+    class LineGraph < Playbook::PbKit::Base
       PROPS = [:configured_class,
```

Comment:
Every kit should have these props defined in the PbKit::Base class:
classname, id, data

`configured_class` should be `configured_classname`
Please reference other kits